tools/portlet_spring_mvc_tmpl/docroot/WEB-INF/jsp/view.jsp

```diff
@@ -0,0 +1,8 @@
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+
+<%@include file="../init.jspf"%>
+
+
```

Comment:
Lets make the default message to be the same as other default messages in other jsps, see the standard mvc for example.